posts/ja/2023-01-31-22.0.0.12_jp.adoc

```diff
@@ -0,0 +1,277 @@
+---
+layout: post
+title: "CVE and many other notable bug fixes in Open Liberty 22.0.0.12"
```

Comment:
`title` needs to be translated from English to Japanese

Reply:
Hi @una-tapa and @takakiyo, I have used machine translation for the proposed Japanese translation. Can you help me check if the translation is correct? Thank you
```suggestion
title: "CVE およびその他の多くの重要なバグ修正 (Open Liberty 22.0.0.12 )"
```

Reply:
I just noticed @takakiyo provided the translation for this sentence in another spot in the blog post. I will use this translation instead of the machine translation.

```suggestion
title: "Open Liberty 22.0.0.12における脆弱性と主なバグの修正"
```